src/FileSystemUtilities.js

```diff
@@ -65,6 +65,8 @@ export default class FileSystemUtilities {
 
   @logger.logifySync
   static unlinkSync(filePath) {
-    fs.unlink(filePath);
+    if (pathExists(filePath)) {
```

Comment:
Looking at this again, I'd say it's better to remove the guard for now.

That would be more consistent with the other `*Sync` methods in FileSystemUtilities, too.